src/test/testing-library/NotificationError.test.tsx

```
@@ -76,39 +77,35 @@ describe('NotificationError Component', () => {
 
     const errorBox: HTMLElement | null = container.querySelector('[aria-live="polite"]');
 
-    if (errorBox) {
-      const computedStyle: CSSStyleDeclaration = window.getComputedStyle(errorBox);
-      expect(computedStyle.display).toBe('flex');
-      expect(computedStyle.flexDirection).toBe('column');
-    }
     expect(errorBox).toBeVisible();
     expect(errorBox).toHaveClass('MuiBox-root');
+
+    const computedStyle: CSSStyleDeclaration = window.getComputedStyle(errorBox!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)